src/analysis/seminorm.lean

```diff
@@ -310,6 +310,24 @@ begin
   exact bot_le,
 end
 
+lemma finset_sup_le_apply (p : ι → seminorm 𝕜 E) (s : finset ι) (x : E) {a : ℝ} (ha : 0 < a)
```

Comment:
```suggestion
lemma finset_sup_apply_le (p : ι → seminorm 𝕜 E) (s : finset ι) (x : E) {a : ℝ} (ha : 0 < a)
```

Reply:
Looks like you applied this rename then reverted it?

Reply:
I was stupid and changed the wrong lemma, I am sorry.